src/lib/auth.js

```diff
@@ -78,3 +77,8 @@ function findUser(profile, accessToken, config, callback)  {
         return callback(false, username + ' is not authorized')
     }
 }
+
+function verifyApiKey(config, req)  {
+    var apiKey = req.headers['apikey'] || req.query['apikey'] || '';
```

Comment:
There's a general concern about putting authorization info in URL params. ie. session ids, API keys etc. However, some systems that you might want to integrate with might only allow you to configure a URL, not add/modify headers. You should always prefer headers for API keys.